app/controllers/application_controller.rb

```diff
@@ -17,6 +18,10 @@ def current_user_id
     warden.user.first.fetch('id')
   end
 
+  def current_user
```

Comment:
We can actually get the current_user name from ```warden.user.first.values_at('first_name', 'last_name')``` which would avoid a query if we wanted. (I imagine we'll need the active record model at some point soon anyway so no big deal). Perhaps memoize the current_user.